lib/beaker/dsl/install_utils/puppet_utils.rb

```diff
@@ -693,7 +693,11 @@ def install_puppetagent_dev_repo( host, opts )
           when /^windows$/
             release_path << 'windows'
             onhost_copy_base = '`cygpath -smF 35`/'
-            arch_suffix = arch =~ /64/ ? '64' : '86'
+            should_install_64bit = host.is_x86_64? && !host['install_32'] && !opts['install_32']
```

Comment:
Can we modify beaker to accept either `:ruby_arch` or `:install_*` (in that order), and deprecate the latter?